docs/src/components/layouts/utility-page-layout/utility-class-table-factory.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+import * as UtilityClassTableComponents from './utility-class-tables';
+
+/**
+ * Gets a utility class table component.
+ *
+ * @param {string} classDisplayComponent component name.
+ * @returns the component.
+ */
+const getUtilityClassTableComponent = (utilityClassDisplayComponent) => {
+	return UtilityClassTableComponents[utilityClassDisplayComponent]?.default;
+};
+
+/**
+ * Factory for Utility layout to dynamically render utility class tables from the
+ * utility-class-tables folder.
+ * @param {*} param.classDisplayComponent The name of the component.
+ * @param {*} param.utilityClasses An array of the utility class information.
+ * @returns
+ */
+const UtilityClassTableFactory = ({
+	utilityClassDisplayComponent,
+	utilityClassDisplayParams,
+}) => {
+	const component = getUtilityClassTableComponent(utilityClassDisplayComponent);
+	// TODO: add use case for no component.
```

Comment:
Can you do this todo please? I would just, IDK, output an alert like we did somewhere else that I can't remember right now.